programs/bpf/rust/bignum/src/lib.rs

```diff
@@ -0,0 +1,129 @@
+//! @brief BigNumber Syscall test
```

Comment:
Be good if the tests also tested errors being returned

Reply:
Will do

Reply:
These unit tests, with harness for additional tests, implemented in `syscalls.rs`